src/Hepteracts.ts

```diff
@@ -489,6 +489,12 @@ export const tradeHepteractToOverfluxOrb = async (buyMax?:boolean) => {
     let toUse: number;
 
     if (buyMax) {
+        if (player.toggles[35]) {
+            const craftYesPlz = await Confirm(`This will attempt to buy as many orbs as possible. \nYou can buy up to ${format(maxBuy, 0, true)} with your hepteracts. Are you sure?`);
```

Comment:
I don't think new lines work in here

Reply:
It actually does, if you have toggle on on, instead of going to next line it will stop right on alert
I was actually surprised that it does work, if to be honest